tests/cupy_tests/core_tests/test_dlpack.py

```diff
@@ -20,6 +20,11 @@ def test_conversion(self, dtype):
         elif cupy.issubdtype(self.dtype, cupy.floating):
             self.array = cupy.random.rand(
                 2, 3).astype(self.dtype)
+        elif cupy.issubdtype(dtype, cupy.complexfloating):
```

Comment:
Nit for consistency :smile:
```suggestion
        elif cupy.issubdtype(self.dtype, cupy.complexfloating):
```

Reply:
Good catch! lol